src/main/java/de/rabea/communication/SocketWriter.java

```diff
@@ -5,6 +5,7 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.io.UnsupportedEncodingException;
```

Comment:
Thanks, I updated all the import statements on master after I raised this PR because I wanted to keep this PR only about the implementation for the Partial Content response. So I'll leave the import statements as they are and once I merge into master, they will be correct.